### PR TITLE
k8s: added memory request/limit for cronjob

### PR DESCRIPTION
**Pull request by bohrasd**
https://theplanttokyo.atlassian.net/browse/SRE-2212

### CODE REVIEW

File: jsonnetlib/k8s.jsonnet [hunk 1]
```diff
@@ -78,6 +78,8 @@ cfg {
     volumes=[],
     podSpec=root.podSpec,
     cronjobSpec=root.cronjobSpec,
+    memoryRequest=root.cronMemoryRequest,
+    memoryLimit=root.cronMemoryLimit,
```
Comment (idealhack) [inline jsonnetlib/k8s.jsonnet:82]: After defining it here, you'll need to use the variable in the function body. Speaking of that, can you add tests to ensure it works as you expect it?

Comment (bohrasd) [reply jsonnetlib/k8s.jsonnet:84]: @idealhack Sorry for the reckless pr. I fixed them and add test case now.

File: jsonnetlib/config.jsonnet [hunk 2]
```diff
@@ -8,6 +8,8 @@ assert std.length(v) > 0 : 'version is empty';
   defaultNamespace: 'default',
   port: 4000,
   baseHost: '',
+  cronCPURequest: '200m',
+  cronCPULimit: '1',
```
Comment (idealhack) [inline jsonnetlib/config.jsonnet:12]: Thanks for adding these.

Considering we are introducing the function with default values at the same time, it's good to set a larger value, we can decrease them after having them set in individual apps. But looking at what we have now with Deployment Pods, I would use smaller values here. How about `50m` and `500m`?

Comment (bohrasd) [reply jsonnetlib/config.jsonnet]: @idealhack Done :)

**Review by idealhack — APPROVED** [review]

**Pull request merged**